lib/constants.ts

```diff
@@ -141,7 +141,10 @@ export const supportedNotificationEvents = new Set([
     's3:ObjectRestore:Post',
     's3:ObjectRestore:Completed',
     's3:ObjectRestore:Delete',
-    's3:LifecycleTransition'
+    's3:LifecycleTransition',
+    's3:LifecycleExpiration:*',
+    's3:LifecycleExpiration:DeleteMarkerCreated',
+    's3:LifecycleExpiration:Delete'
```

Comment:
please add the trailing comma : so that new lines can be added without modifying the existing ones

```suggestion
    's3:LifecycleExpiration:Delete',
```